app/controllers/admin/categories_controller.rb

```diff
@@ -3,11 +3,19 @@ class CategoriesController < Admin::BaseController
     before_action :set_category, only: %i[show edit destroy update]
 
     def index
-      @pagy, @category = pagy(Category.all, items: 5)
+      @pagy, @categories = pagy(Category.all, items: 5)
       respond_to do |format|
         format.html
         format.csv { send_data Category.all.to_csv, filename: "categories-#{Date.today}.csv" }
       end
+
+      if params[:search].present?
```

Comment:
Can we refactor this in such a way where we always call the search method and it early returns the all categories if the search param is blank? In that way first, we can avoid conditional here. Secondly order line will be used once as well.
Lastly, the order line is quite long. Make a conditional block for that.